js/compiler/igor-controller.js

```diff
@@ -72,20 +79,43 @@ function job_remove(job) {
  * Select the flags for Igor to run the job.
  * @param {GMLProject} project
  * @param {string} runtime_path
+ * @param {string?} user_path
  * @param {IgorSettings} settings
  * @returns {Result<string[]>}
  */
-function job_flags_get(project, runtime_path, settings) {
+function job_flags_get(project, runtime_path, user_path, settings) {
+
+    /**
+     * @type {{[K in IgorPlatform]?: string}}
+    */
+    const exts = {
+        Windows: ".zip",
+        Mac: ".zip",
+        Linux: ".appimage",
+    };
+
+    let projectName = $gmedit['gml.Project'].current.name;
+    // Remove .yyp file extension
+    projectName = projectName.substring(0, projectName.length - 4);
```

Comment:
I'd prefer steering away from removing extensions and stuff with magic numbers, I _think_ node's fs module has something to do this for you? We've already got that imported for other path stuff so would be better to apply it here.

Reply:
solved by just doing `project.displayName` instead of `project.name` and removing the .yyp file extension